iac/provider-gcp/nomad/jobs/orchestrator.hcl

```diff
@@ -5,23 +5,47 @@ job "orchestrator-${latest_orchestrator_job_id}" {
   priority = 90
 
   group "client-orchestrator" {
+    network {
+      mode = "host"
+
+      port "api" {
+        // todo: remove this once all API and client-proxy jobs
+        //  can pull the port number from nomad.
+        static = "${port}"
+      }
+
+      port "proxy" {
+        // todo: remove this once all API and client-proxy jobs
+        //  can pull the port number from nomad.
+        static = "${proxy_port}"
+      }
+    }
+
     service {
-      name = "orchestrator"
-      port = "${port}"
+      name     = "orchestrator"
+      port     = "api"
+      provider = "nomad"
```

Comment:
This enables the health checks to work. Without it, we default to `consul`, which doesn't function.